app/controllers/planning_application/consultations_controller.rb

```diff
@@ -58,6 +58,8 @@ def destroy
     def send_neighbour_letters
       return if @consultation.blank?
 
+      @planning_application.send_neighbour_consultation_letter_copy_mail
```

Comment:
I wonder if we should only send a copy if we know the letters have sent? But not sure about it

Reply:
Good point.
Feels almost as if everything needs to be a success or nothing at all.
But I could probs move this to below the letter sending.